nss-cohort-36/bangazon-api-virtual_magicians
# Sp shopping cart

# Description
Created Order and OrderProduct views in order to allow user to view Shopping Cart.
**You will have to load new fixtures files (see below for instructions)**
Closes [PART OF] https://github.com/nss-cohort-36/bangazon-api-virtual_magicians/issues/2

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-shopping-cart`
4. `python manage.py makemigrations virtualmagicians`
5. `python manage.py migrate`
6. `python manage.py runserver`
```
********************************************************
Part 2 if you haven't installed the new fixture files
********************************************************
python manage.py loaddata paymenttype
python manage.py loaddata order
python manage.py loaddata orderproduct
```

# Checklist:
